src/client/rsg-components/StyleGuide/StyleGuide.js

```diff
@@ -74,6 +81,24 @@ export default class StyleGuide extends Component {
 		});
 	}
 
+	getDefaultTheme() {
+		const {
+			config: { defaultTheme, themes },
+		} = this.props;
+
+		if (defaultTheme) {
+			return defaultTheme;
+		}
+
+		return themes && themes[0] && themes[0].id;
+	}
+
+	handleThemeSwitch(theme) {
```

Comment:
Just a preference:

```suggestion
	handleThemeChange = currentTheme => {
```

And then `this.setState({currentTheme})` and `onThemeSwitch={this.handleThemeChange}`.

Reply:
Yeah, that's nicer.